src/Util/CacheComponentInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Mongolid\Util;
+
+/**
+ * CacheComponentInteface describes the API of an CacheComponent that may be
+ * used with Mongolid.
+ */
+interface CacheComponentInterface
+{
+    /**
+     * Retrieve an item from the cache by key.
+     *
+     * @param string $key cache key of the item to be retrieved
+     *
+     * @return mixed
+     */
+    public function get(string $key);
+
+    /**
+     * Store an item in the cache for a given number of minutes.
+     *
+     * @param string $key     cache key of the item
+     * @param mixed  $value   value being stored in cache
+     * @param float  $minutes cache ttl
+     */
+    public function put(string $key, $value, float $minutes);
```

Comment:
```suggestion
    public function put(string $key, $value, float $seconds);
```
There are differences between `mongolid` and `mongolid-laravel`. In `mongolid` the cache is handled in minutes and `mongolid-laravel` is handled in seconds.

I think we could use the same behavior on both.

Reply:
This part of cache is actually a rollback. Cause in the v3 doesnt have cache.
If in the future we decide to keep with cache, will considerate to make this change.